unit_tests/glhe/profiles/test_external_load.py

```diff
@@ -12,12 +13,12 @@
 class TestExternalLoad(unittest.TestCase):
 
     @staticmethod
-    def add_instance(path):
+    def add_instance(path: Path):
         d = {'fluid': {'fluid-type': 'water'},
-             'load-profile': [{'load-profile-type': 'external', 'name': 'my name', 'path': path}]}
+             'load-profile': [{'load-profile-type': 'external', 'name': 'my name', 'path': str(path)}]}
 
-        temp_dir = tempfile.mkdtemp()
-        temp_file = os.path.join(temp_dir, 'temp.json')
+        temp_dir = Path(tempfile.mkdtemp())
+        temp_file = temp_dir / 'temp.json'
```

Comment:
There were quite a few places that things like this were repeated.  We could add a unit test fixture to collect a bunch of these.  It would be nice to move the `unit_tests` folder inside the glhe code, that way the fixture import would look like `from glhe.tests.fixture import GLHEFixture`, rather than `from unit_tests ...` like it would be right now.